bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -258,9 +258,24 @@
    * If timeouts are set, how many milliseconds should pass before a DEADLINE_EXCEEDED for a long
    * read? Currently, this feature is experimental.
    */
+  @Deprecated
```

Comment:
please add to the javadoc what to use instead

Reply:
```java
  /**
   *  @deprecated Please use {@link #BIGTABLE_MUTATE_RPC_TIMEOUT_MS_KEY} or {@link
   *     #BIGTABLE_READ_RPC_TIMEOUT_MS_KEY} based on long operation.
   */
```

I believe this deprecated comment should be enough for the user to be able to distinguish new timeout key.